components/services/bat_ledger/bat_ledger_impl.h

```diff
@@ -33,7 +33,7 @@ class BatLedgerImpl : public mojom::BatLedger,
   void CreateWallet(const std::string& safetynet_token,
       CreateWalletCallback callback) override;
   void FetchWalletProperties(FetchWalletPropertiesCallback callback) override;
-  
+
```

Comment:
remove this change

Reply:
It was lint error, why remove it?

Reply:
I want to reduce changes as much as possible. So removing things like this would reduce PR size

Reply:
I was told that lint fixes are must have. As it will fail ci otherwise.

Reply:
but new line here will not be a lint problem

Reply:
Ya, sorry, I thought there were spaces removed. I'll remove it.

Reply:
I'm not sure what is this change. We have identical code here with master
https://github.com/brave/brave-core/blob/91cd80d5a17a44b509ca72b60abe7c6050a67d48/components/services/bat_ledger/bat_ledger_impl.h#L35
https://github.com/brave/brave-core/blob/32feb76f904597f554c74beebacce9026ce56f29/components/services/bat_ledger/bat_ledger_impl.h#L36
So this PR shouldn't bring anything new here.